src/main/java/org/broadinstitute/hellbender/tools/exome/CalculateCoverageComponents.java

```diff
@@ -80,6 +81,9 @@
 
     @Override
     protected void runPipeline(final JavaSparkContext ctx) {
+        if (! new HDF5Library().load(null)){
+            throw new UserException.HardwareFeatureException("Cannot load the required HDF5 library");
```

Comment:
i will improve the message. maybe it is possible. i dont think we should mention it unless we've tested that it works. I'm just going to say `Cannot load the required HDF5 library. HDF5 is currently supported on x86-64 architecture and Linux or OSX systems.`